clusterdock/topologies/cdh/actions.py

```diff
@@ -162,18 +198,48 @@ def start(args):
                 deployment.cluster.delete_service(service.name)
 
     hue_server_host_port = get_host_port_binding(primary_node.container_id, HUE_SERVER_PORT)
+    is_hue_installed = False
```

Comment:
This pattern is kinda gross. Is there any way to do this without a state variable?